db/seeds.rb

```diff
@@ -49,8 +49,8 @@ def set_time
 def set_route(airports_id)
   arrival_id = airports_id.sample
   departure_id = airports_id.sample
-  if departure_id == arrival_id
-    arrival_id = airports_id.sample
+  until departure_id != arrival_id do
```

Comment:
Do not use `do` with multi-line `until`.<br>Favor modifier `until` usage when having a single-line body.